### PR TITLE
Connector and models for get financial details

### PR DESCRIPTION
**Pull request by chandraAndem**

### CODE REVIEW

File: app/models/financialDetails/Charge.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                  `type`: Option[String],
+                  totalAmount: Option[BigDecimal],
+                  outstandingAmount: Option[BigDecimal],
+                  subItem: Option[Seq[SubItem]]
+                 )
+
+object Charge {
+
+  implicit val format: Format[Charge] = Json.format[Charge]
+}
```
Comment (JSimmons072019) [inline app/models/financialDetails/Charge.scala:33]: needs a new line at the end of the file

File: app/models/financialDetails/SubItem.scala [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                   outgoingPaymentMethod: Option[String],
+                   paymentAmount: Option[BigDecimal],
+                   dueDate: Option[String],
+                   paymentMethod: Option[String],
+                   paymentId: Option[String])
+
+object SubItem {
+
+  implicit val format: Format[SubItem] = Json.format[SubItem]
+}
```
Comment (JSimmons072019) [inline app/models/financialDetails/SubItem.scala:34]: needs a new line at the end of the file

File: test/assets/ChargesTestConstants.scala [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  val testChargesErrorModelParsing: ChargesErrorModel = ChargesErrorModel(
+    testErrorStatus, "Json Validation Error. Parsing Charges Data Response")
+
+  val testChargesErrorModel: ChargesErrorModel = ChargesErrorModel(testErrorStatus, testErrorMessage)
+  val testChargesErrorModelJson: JsValue = Json.obj(
+    "code" -> testErrorStatus,
+    "message" -> testErrorMessage
+  )
+}
```
Comment (JSimmons072019) [inline test/assets/ChargesTestConstants.scala:107]: needs a new line at the end of the file

File: test/assets/PaymentsTestConstants.scala [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+  val testPaymentsErrorModelParsing: PaymentsErrorModel = PaymentsErrorModel(
+    testErrorStatus, "Json Validation Error. Parsing Payments Data Response")
+
+  val testPaymentsErrorModel: PaymentsErrorModel = PaymentsErrorModel(testErrorStatus, testErrorMessage)
+  val testPaymentsErrorModelJson: JsValue = Json.obj(
+    "code" -> testErrorStatus,
+    "message" -> testErrorMessage
+  )
+}
```
Comment (JSimmons072019) [inline test/assets/PaymentsTestConstants.scala:103]: needs a new line at the end of the file

Comment (chandraAndem) [reply test/assets/PaymentsTestConstants.scala]: Done

File: test/assets/ChargesTestConstants.scala (hunk 3 above)
Comment (chandraAndem) [reply test/assets/ChargesTestConstants.scala]: Done

File: app/models/financialDetails/SubItem.scala (hunk 2 above)
Comment (chandraAndem) [reply app/models/financialDetails/SubItem.scala]: Done

File: app/models/financialDetails/Charge.scala (hunk 1 above)
Comment (chandraAndem) [reply app/models/financialDetails/Charge.scala]: Done

**Pull request merged**